scilpy/io/utils.py

```diff
@@ -13,6 +13,14 @@
 from scilpy.utils.bvec_bval_tools import DEFAULT_B0_THRESHOLD
 
 
+def check_tracts_same_format(parser, filename_list):
+    _, ref_ext = os.path.splitext(filename_list[0])
+
+    for filename in filename_list[1:]:
+        if not filename is None and not os.path.splitext(filename)[1] == ref_ext:
```

Comment:
can we change the first part with something like this: isinstance(filename, str)